appveyor.yml

```diff
@@ -36,3 +36,29 @@ build_script:
     & Setup\MakePortableArchive.cmd Release %APPVEYOR_BUILD_VERSION%
     #Upload a portable archive, not a installer
     Get-ChildItem Setup\GitExtensions-Portable-*.zip | % { Push-AppveyorArtifact $_.FullName -FileName $_.Name }
+
+test_script:
+- ps: |
+    $testAssemblies = @(
+        "UnitTests\GitCommandsTests\bin\Release\GitCommandsTests.dll"
+        "UnitTests\GitUITests\bin\Release\GitUITests.dll"
+        "UnitTests\GravatarTests\bin\Release\GravatarTests.dll"
+        "UnitTests\Plugins\ReleaseNotesGeneratorTests\bin\Release\ReleaseNotesGeneratorTests.dll"
+        "UnitTests\ResourceManagerTests\bin\Release\ResourceManagerTests.dll"
```

Comment:
Can we do a wildcard, something like `UnitTests\**\bin\Release\*Tests.dll`?

Reply:
I imagine it's possible. For this pull request, I just started with the list of test assemblies reported by one of the existing AppVeyor master branch builds to define this array. Would you be comfortable with leaving the change as a follow-up item?

Reply:
Sure, nw